include/mesh/CoupledMFEMMesh.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+#include "ExclusiveMFEMMesh.h"
+
+/**
+ * CoupledMFEMMesh
+ */
+class CoupledMFEMMesh : public ExclusiveMFEMMesh {
+ public:
+  static InputParameters validParams();
+
+  CoupledMFEMMesh(const InputParameters& parameters);
+
+  virtual void buildMesh() override;
+
+  // Builds the corresponding MOOSE mesh
+  void buildRealMesh();
```

Comment:
Keen to rename `buildRealMesh` to avoid confusion when solving for complex variables.